starky/src/cross_table_lookup.rs

```diff
@@ -324,12 +328,14 @@ pub(crate) fn cross_table_lookup_data<'a, F: RichField, const D: usize, const N:
     constraint_degree: usize,
 ) -> [CtlData<'a, F>; N] {
     let mut ctl_data_per_table = [0; N].map(|_| CtlData::default());
-    for CrossTableLookup {
-        looking_tables,
-        looked_table,
-    } in cross_table_lookups
-    {
-        log::debug!("Processing CTL for {:?}", looked_table.table);
+    for CrossTableLookup { looking_tables } in cross_table_lookups {
+        log::debug!(
+            "Processing CTL for {:?}",
+            looking_tables
+                .iter()
+                .map(|table| table.table)
+                .collect::<Vec<_>>()
+        );
```

Comment:
It seems like printing all the involved tables will look pretty heavy compared to the previous display. Especially as we can have multiple looking columns from the same table.

Reply:
Alternatively, as we always have the `looked` table being pushed last, we could either get back to the previous display as 
```suggestion
        log::debug!(
            "Processing CTL for table {:?}",
            looking_tables
                .last()
                .expect("We should always have tables")
                .table
        );
```

or have a more complete one also listing the `looking` ones (although from my point above, I find it meaningless to see that table `N` appears 50 times, if we can't print more specific information as to what we are looking for).

Reply:
So in our use of CTL, we often don't have a single table that's somehow special that it would make to single out and sense to print here.

If you are afraid of having one table appear 50 times, we can consolidate repeated use of the same table and replace that with a count (as a number).

Reply:
Yeah, at least counting the occurences instead of printing all tables would be better (in our case we do have some CTLs with a fairly large number of looking tables).

> So in our use of CTL, we often don't have a single table that's somehow special

Fair, for logging it isn't super useful compared to the debugging function.

Reply:
I'm keeping counts now.